types/ramda/index.d.ts

```diff
@@ -30,17 +30,16 @@
 //                 Nicholai Nissen <https://github.com/Nicholaiii>
 //                 Mike Deverell <https://github.com/devrelm>
 //                 Jorge Santana <https://github.com/LORDBABUINO>
+//                 Mikael Couzic <https://github.com/couzic>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.7
+// TypeScript Version: 4.2
 
 import * as _ from "ts-toolbelt";
 import {
-    Arity0Fn,
     Arity1Fn,
     Arity2Fn,
     AssocPartialOne,
     ComposeWithFns,
-    Dictionary,
```

Comment:
These changes to imports doesn't seem relevant to your PR.

Reply:
Yes you're right, here I was just applying the boy scout rule by removing some old unused imports. I can revert that change if you want, I don't mind much either way.

Reply:
You're right, it **is** a good rule of thumb, I just think it deserves it's own PR, so conversation about the changes can happen where relevant. I do believe this specific import is kept in case zipObj would use it again, there's a TODO for it.